samples/charger_and_events/README.rst

```diff
@@ -0,0 +1,108 @@
+.. _npmx_charger_and_events_sample:
+
+npmx: CHARGER and EVENTS
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The npmx charger and events sample demonstrates how to control PMIC charger and receive events using npmx drivers.
```

Comment:
```suggestion
The charger and events sample demonstrates how to control charger and receive events using npmx drivers.
```

Reply:
@masz-nordic @wilu-nordic 
Marcin, can we keep "PMIC" acronym just for clarity? As this is a very first intro sentence, would be great if we can be as specific as possible and use proper names at least at the beginning of this page.

Reply:
Okay